db/seeds.rb

```diff
@@ -4,21 +4,30 @@
 # Examples:
 #
 
-if User.where(user_name:'Admin').first.blank?
-  harvester = User.new(user_name: 'Admin',
-                       email: Settings.admin_user.email,
-                       password: Settings.admin_user.password)
-  harvester.roles = [:admin]
-  harvester.skip_confirmation!
-  harvester.save!(validate: false)
+if User.where(user_name: 'Admin').first.blank?
+  admin = User.new
+else
+  admin = User.where(user_name: 'Admin').first
```

Comment:
I changed this mainly to help me setup a database which I had copied off staging. My logic follows

- seed is only run at initial db:setup at the moment
- the password seed may not be the same, meaning logins won't work. Useful to reset these accounts
- changing the behavior makes db:seed useful for setting the state of the database to a desired state. Admin/harvester should always have the email and password from settings

I thought these changes were useful to keep. Main reason for this whole PR :smile: